libafl/src/feedbacks/map.rs

```diff
@@ -634,7 +658,9 @@ where
         let initial = observer.initial();
         if interesting {
             let len = history_map.len();
-            let filled = history_map.iter().filter(|&&i| i != initial).count();
+            let covered = history_map
```

Comment:
i forgot to delete but this if interesting branch should be deleted

Reply:
Are you sure? That means no coverage stats if you're using the vector version.

Reply:
Oh, do you mean it should be unconditional rather than behind an if-statement?

Reply:
nono they will be sent in another function append_metadata.
we should not send this event in is_interesting